### PR TITLE
Starting on the Create Encounter flow

### PR DESCRIPTION
**Pull request by clayplumridge**

### CODE REVIEW

**Review by grundygolem2 — CHANGES_REQUESTED** [review]
No idea why, but I get a scrollbar at the bottom of the +encounters view. plz no.

Also, for this level of PR, can we get a "cancel create" and a delete button on encounters?

File: src/render/components/TextField.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    makeStyles,
+    TextField as MuiTextField,
+    TextFieldProps as MuiTextFieldProps
+} from "@material-ui/core";
+import { css } from "@/util";
+import { IObservableValue } from "@/render/core/Observable";
+import { Observer } from "./Observer";
+
+export interface TextFieldProps
+    extends Omit<MuiTextFieldProps, "onChange" | "value"> {
```
Comment (grundygolem2) [inline src/render/components/TextField.tsx:13]: What is Omit?

**Comment by clayplumridge** [comment]
> No idea why, but I get a scrollbar at the bottom of the +encounters view. plz no.

Can you grab a SS? @grundygolem2 

> Also, for this level of PR, can we get a "cancel create" and a delete button on encounters?

Yeah should be doable

File: src/render/views/EncounterManagementView.tsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ export const EncounterManagementView: React.FC<{}> = () => {
 const useItemStyles = makeStyles(theme =>
     createStyles({
         gridItemPaper: {
-            padding: theme.spacing(2),
+            padding: theme.spacing(1),
```
Comment (grundygolem2) [inline src/render/views/EncounterManagementView.tsx:76]: y is this the fix, gross, but w/e

**Review by grundygolem2 — APPROVED** [review]

File: src/render/views/EncounterManagementView.tsx (hunk 2 above)
Comment (clayplumridge) [reply src/render/views/EncounterManagementView.tsx:76]: Not sure tbh.

File: src/render/components/TextField.tsx (hunk 1 above)
Comment (clayplumridge) [reply src/render/components/TextField.tsx:13]: It's the first type parameter minus the fields from the second type parameter, for the purposes of clear overriding in this case

**Pull request merged**